backend/src/apiserver/filter/filter.go

```diff
@@ -207,6 +209,86 @@ func replaceMapKeys(m map[string][]interface{}, keyMap map[string]string, prefix
 	return nil
 }
 
+func (f *Filter) FilterK8sPipelines(pipeline v2beta1.Pipeline) (bool, error) {
+	if len(f.eq) > 0 {
+		return false, util.NewInvalidInputError("Filter eq is not implemented for Kubernetes native API. Only substring is supported.")
```

Comment:
Please make this change in the other error messages as well. Also, consider making this a template string that just replace the filter name so you don't have to duplicate the text every time.
```suggestion
		return false, util.NewInvalidInputError("Filter eq is not implemented for the Kubernetes pipeline store. Only substring is supported.")
```